packages/tools/release-utils-internal/src/vars.ts

```diff
@@ -0,0 +1,10 @@
+// As env var is alays string, we expect it to be "true",
+// converting eny other value to false.
+export const skipDepsBuild = (): boolean => process.env.MX_TOOLS_SKIP_DEPS_BUILD === "true";
+
+// NOTE: this is not a JS regex but C# and it passed as flag.
+export const mxutilExcludeFilesRegEx = (): string => {
+    return process.env.MX_TOOLS_EXCLUDE_FILES_ON_EXPORT ?? "^(resources|userlib)/.*";
+};
+
+export const isDebug = (): boolean => typeof process.env.DEBUG !== "undefined";
```

Comment:
I think we could just make it a constant instead of a function.

Reply:
I used function to make it more dynamic. This give script also ability to set `process.env.DEBUG`

Reply:
I don't see this being used anywhere at the moment, also `init.ts` always evaluates `isDebug` at the beginning making it impossible to change dynamically.